### ISSUE TITLE
Function to get sensor's "device information"

### ISSUE THREAD

**Issue by skgrange** [dobra-dobra/Python-Sensirion-SPS30#1]
Hello Szymon,
Great work on the package, it has helped us out with our work here. 

If you still have a sensor and are willing, could I ask that another function is developed to read the sensor's device information (page 16 of the sensor's datasheet). We would like to use the serial number in our logging programmes as unique identifier for the exported files. Many thanks.

**Comment by szajakubiak** [comment]
Hi Stuart,
it's nice to hear that it was useful for someone. I don't have a lot of time for that right now, but I will try to look at it till end of this week. Although don't expect miracles - I'm scientist not professional programmer.
Can you share any info about your project? I'm also interested in these low-cost air quality sensors, although I'm more focused on workplace exposure assessment. Moreover, I spent some time at Eawag during my PhD and had some cooperation with Empa.
Best regards
Szymon Jakubiak

**Comment by szajakubiak** [comment]
Please check if this new function which I just added work for you. I have two of this sensors and it does return different values, although I'm not sure how serial number should look like.

**Comment by skgrange** [comment]
Hello Szymon,
Many thanks for the very quick work. I have implemented the changes in my programmes ([here](https://github.com/skgrange/py_sensirion_sps30_logger)) so that is good. Like you, I am not too sure what the serial number should look like, but it seems like a unique identifier so I have used it for now. 

There is a small project here with creating a small sensor package with the Sensirion sensor and a Raspberry Pi. Things seem to be working well so they will deployed near the office for some testing. I am not too sure what the goal is, maybe people are just looking at the sensors which are available and if they are useful? Thanks again.
